docs/versioned-plugins/codecs/compress_spooler-v2.0.5.asciidoc

```diff
@@ -20,7 +20,7 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-
+The compress_spooler codec.
```

Comment:
I've added text to get this to work, but this needs to be replaced with a real description. None of the versions have descriptions.